save-cloud-common/src/commonMain/kotlin/org/cqfn/save/agent/TestExecutionDto.kt

```diff
@@ -22,4 +23,5 @@ data class TestExecutionDto(
     val startTimeSeconds: Long?,
     val endTimeSeconds: Long?,
     val testSuiteName: String? = null,
+    val tags: List<String>? = null,
```

Comment:
```suggestion
    val tags: List<String> = emptyList(),
```
One of the reasons I suggested collection is that you can make it non-nullable

Reply:
Yes, good advise, fixed